assets/js/modules/ads/components/settings/SettingsView.js

```diff
@@ -57,9 +59,37 @@ export default function SettingsView() {
 
 	const isPaxView = paxEnabled && ( paxConversionID || extCustomerID );
 
+	const isConversionTrackingEnabled = useSelect( ( select ) =>
+		iceEnabled ? select( CORE_SITE ).isConversionTrackingEnabled() : false
+	);
+
+	const conversionTrackingSettingValue = isConversionTrackingEnabled
+		? __( 'Enabled', 'google-site-kit' )
+		: __( 'Disabled', 'google-site-kit' );
+
 	return (
 		<Fragment>
 			<AdBlockerWarning />
+			{ iceEnabled && (
+				<div className="googlesitekit-settings-module__meta-item">
+					{
+						// This is required to add correct spacing if the <AdBlockerWarning> component is present.
+						isAdBlockerActive && <br />
+					}
+					<h5 className="googlesitekit-settings-module__meta-item-type">
+						{ __(
+							'Enhanced conversion tracking',
+							'google-site-kit'
+						) }
+					</h5>
+					<p className="googlesitekit-settings-module__meta-item-data">
+						<DisplaySetting
+							value={ conversionTrackingSettingValue }
```

Comment:
`conversionTrackingSettingValue` seems to be only used here, we can skip defining extra variable, and handle the label render directly here

```suggestion
                                                        value={isConversionTrackingEnabled
		? __( 'Enabled', 'google-site-kit' )
		: __( 'Disabled', 'google-site-kit' )
}
```